chapters/OOPs!/chapter.md

```diff
@@ -1,92 +1,87 @@
-#Ooops! = Object Oriented Programming + Classes#
+# Ooops! = Object Oriented Programming + Classes
 
 
-##Overview
+## Overview
 This tutorial is a quick and practical introduction to Object Oriented Programming in openFrameworks and a how-to guide to build and use your own classes.
 By the end of this chapter you should understand how to create your own objects and have a lot of balls bouncing on your screen!
 
 ![balls screenshot](images/balls_screenshot.png "balls screenshot")
 
-##What is OOP
-Object Oriented Programming is a programming paradigm based on the use of objects and their interactions. A recurring analogy is to see a "Class" as a cookie cutter that can create many cookies, "the Objects".
+## What is Object Oriented Programming
+Object Oriented Programming (OOP) is a programming paradigm based on the use of objects and their interactions. A recurring analogy is to see a "class" as a cookie cutter that can create many cookies, the "objects".
 Some terms and definitions used within OOP are listed below:
 
--A Class defines the characteristics of a thing - the object - and its behaviors; it defines not only its properties and attributes but also what it can do.
+- A class defines the characteristics of a thing - the object - and its behaviours; it defines not only its properties and attributes but also what it can do.
 
--An Object is an instance of a class.
+- An object is an instance of a class.
 
--The Methods are the objects abilities
+- The methods are the objects abilities.
 
-##How to build your own Classes (simple Class)
-Classes and objects are similar to the concepts of movie clips and instances in Flash and are also a fundamental part of Java programming. 
+## How to build your own Classes (simple Class)
+~~Classes and objects are similar to the concepts of movie clips and instances in Flash and are also a fundamental part of Java programming.~~ **[t: What value does this sentence add? (I ask because I doubt beginning oF coders know the details of Flash. The statement about Java is also true for Python etc.)]**
```

Comment:
done! : )

_____________ ____________ _________ _____ ___ __ _ _
http://www.rux-werx-here.net + http://www.vimeo.com/rux +
twitter: @rux_twitts_here

On Fri, May 1, 2015 at 5:52 PM, tpltnt <notifications@github.com> wrote:

> In chapters/OOPs!/chapter.md
> <https://github.com/openframeworks/ofBook/pull/139#discussion_r29536492>:
>
> >
> > -##How to build your own Classes (simple Class)
> > -Classes and objects are similar to the concepts of movie clips and instances in Flash and are also a fundamental part of Java programming.
> > +## How to build your own Classes (simple Class)
> > +~~Classes and objects are similar to the concepts of movie clips and instances in Flash and are also a fundamental part of Java programming.~~ **[t: What value does this sentence add? (I ask because I doubt beginning oF coders know the details of Flash. The statement about Java is also true for Python etc.)]**
>
> @ruxrux <https://github.com/ruxrux> what about this sentence? Remove it
> like I proposed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openframeworks/ofBook/pull/139/files#r29536492>.
>